src/app/loginComponents/accounts/account-sidebar/account-sidebar.component.ts

```diff
@@ -8,20 +8,26 @@ import { Profile, User } from '../../../shared/swagger';
 import { UsersService } from '../../../shared/swagger/api/users.service';
 import { UserQuery } from '../../../shared/user/user.query';
 import { UserService } from '../../../shared/user/user.service';
+import { GravatarService } from '../../../gravatar/gravatar.service';
 
 @Component({
-  selector: 'app-accounts-internal',
-  templateUrl: './accounts.component.html',
-  styleUrls: ['./accounts.component.scss'],
+  selector: 'app-account-sidebar',
+  templateUrl: './account-sidebar.component.html',
+  styleUrls: ['./account-sidebar.component.scss'],
 })
-export class AccountsInternalComponent implements OnInit {
+export class AccountSidebarComponent implements OnInit {
   user: User;
   TokenSource = TokenSource;
   googleProfile: Profile;
   gitHubProfile: Profile;
   hasGitHubToken$: Observable<boolean>;
   hasGoogleToken$: Observable<boolean>;
   public isRefreshing$: Observable<boolean>;
+  public syncBadgeGit: boolean = false;
+  public syncBadgeGoogle: boolean = false;
+  public githubTable: string[] = ['name', 'email', 'company', 'location', 'bio'];
```

Comment:
This is very JavaScript-y, not so much TypeScript-y. :) The compiler can no longer enforce that we are using the correct property names (you currently are, but properties could get added/deleted/modified, even though unlikely in this particular case).

There are trade-offs on dynamic vs. typed languages, but we've gone with typed on this project, so...

1. Live with it, but I would at least change the property name and/or add a comment saying these are property names of the User class and must match.
2. Change the HTML to use the typed names. I know it's more verbose and a bit of hassle, but that's one of the tradeoffs on using a typed language. :)

I lean towards 2, but curious what other reviewers think.

Reply:
> If you prefer 2 I can update it, what do you mean by changing the html to use the typed names? Like access the data with the names individually like github.name?

Yes, I mean that. Sorry, I know it's a little bit more verbose, but then the compiler can check things.